components/Filter/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+
+const Filter = ({ results, setFilteredResults }) => {
+  const [filterText, setFilterText] = useState("");
+
+  const handleFilterChange = (e) => {
+    const newFilterText = e.target.value;
```

Comment:
There is no need to be brief here, you can write `event` to make clear what's happening.